lib/matplotlib/patheffects.py

```diff
@@ -4,10 +4,13 @@
 and :class:`~matplotlib.patches.Patch`.
 """
 
+from math import radians, sin, cos, ceil, sqrt
```

Comment:
I would suggest just using the (prefixed) functions from numpy, which work fine here (and the performance difference is irrelevant).